website/docs/cdktf/create-and-deploy/aws-adapter.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-page_title: "[technical preview] AWS Adapter"
+page_title: [technical preview] AWS Adapter - CDK for Terraform
```

Comment:
Vercel fails without these (as the `[` is interpreted by yaml as opening an array)
```suggestion
page_title: "[technical preview] AWS Adapter - CDK for Terraform"
```